Patches.py

```diff
@@ -2022,21 +2139,57 @@ def read_rom_item(rom, item_id):
     row = item_row_struct.unpack(row_bytes)
     return { item_row_fields[i]: row[i] for i in range(len(item_row_fields)) }
 
-
 def write_rom_item(rom, item_id, item):
     addr = rom.sym('item_table') + (item_id * item_row_struct.size)
     row = [item[f] for f in item_row_fields]
     row_bytes = item_row_struct.pack(*row)
     rom.write_bytes(addr, row_bytes)
 
-
+texture_struct = struct.Struct('>HBxxxxxII') # Match item_row_t in item_table.h
+texture_fields = [
+    'texture_id', 'file_buf', 'file_vrom_start', 'file_size',
+    ]
+
+def read_rom_texture(rom, texture_id):
+    addr = rom.sym('texture_table') + (texture_id * texture_struct.size)
+    row_bytes = rom.read_bytes(addr, texture_struct.size)
+    row = texture_struct.unpack(row_bytes)
+    logger = logging.getLogger('')
+    logger.info(texture_struct.size)
+    logger.info(row)
+    return { texture_fields[i]: row[i] for i in range(len(texture_fields)) }
+
+def write_rom_texture(rom, texture_id, texture):
+    addr = rom.sym('texture_table') + (texture_id * texture_struct.size)
+    row = [texture[f] for f in texture_fields]
+    row_bytes = texture_struct.pack(*row)
+    
+    logger = logging.getLogger('')
+    logger.info(texture_struct.size)
+    logger.info(texture)
+    rom.write_bytes(addr, row_bytes)
```

Comment:
I've scrolled past a lot of them already, but these should be changed to logger.debug before merging

Reply:
Think most of this has been done. But I plan on removing pretty much all of the logging that I've added once I'm ready to mark the PR as ready. It was mostly there to facilitate testing and development and doesn't need to exist anymore.